src/cpp/shared/adventure/map.h

```diff
@@ -378,6 +378,13 @@ enum ADVENTURE_MAP_LOCATION {
   TILE_HAS_EVENT = 128,
 };
 
+
+enum barriercolors {
```

Comment:
I'm pretty sure these colors are the same as the player colors, no need.

*checks*

char *xBarrierColor[] =
{
  "aqua",
  "blue",
  "brown",
  "gold",
  "green",
  "orange",
  "purple",
  "red"}

Oh. Good job. Add the others.